### PR TITLE
Added some info to the page

### PR DESCRIPTION
**Pull request by Skyhero-admin**
# Pull Request

It would be good to get some feedback on the current information for the landing page
I've some info in the index.html file here https://github.com/Skyhero-admin/homepage

If you have any suggestions to add or if something doesn't seem right and should be removed then please do let me know 😅

Also It's still not styled yet so I need some ideas for it

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,13 +13,42 @@
...
+      </h3>
+      <p>
+        <br>
+        Participate in our community projects
+        <br>
+        Of course this is a Programming focused community, but thats not all that we're about
+        <br>
+        <b>We also have a Gaming discussion channel on our Discord Server</b>Where our community members stream their Gameplays quite often. We also play together with the other community members quite often.
+        <br>
+        <b>We also have a lot of other fun activity's happenning every now then</b>
```
Comment (nhcarrigan) [inline index.html:32]: We should not use `<br>` tags to create spacing as this is an accessibility concern.

File: index.html [hunk 2]
```diff
@@ -13,13 +13,42 @@
...
-      our community members, so come back soon to see what they've accomplished!
-    </p>
+    <header>
+      <h1>Welcome to Carrigan's Community!</h1>
+    </header>
+    <main>
+    <section id='opening'>
+      <h3>
+        <b>Our community</b> tries to help people on their programming Journey and also gives you chance to know what it feels like to be a part of a community with all sorts of people from all around the world .
+      </h3>
```
Comment (nhcarrigan) [inline index.html:23]: I'm not sure this should be an `h3`.

File: index.html [hunk 3]
```diff
@@ -13,13 +13,42 @@
...
+      <p></p>
+      
+    </section>
+    <section id="whoisNH">
+      <h2>Who is Nhcarrigan ?</h2>
+      <p>I am a web developer focusing on the MEAN stack with a love for TypeScript. Development is a passion of mine, which began with the FreeCodeCamp curriculum on 15 April 2020. In December of 2020 I accepted a position with freeCodeCamp, and have continued to learn, grow, and acquire new skills.
+      <br>
+      In my free time I enjoy playing a video game or reading a book. I enjoy learning new skills and picking up random bits of trivia. For example, did you know that an "emordnilap" is a word which when reversed spells another word?
+      <br>
+      I am a firm believer in open-source. All of my project files are available on GitHub, and I am open to Pull Requests and suggestions on all of them. If you would like to contribute, you can access my GitHub profile on the "connect" page.</p>
```
Comment (nhcarrigan) [inline index.html:46]: Again, we want to avoid `br` tags.

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
This is looking good - just a couple of nitpicks. 🙂

File: index.html (hunk 1 above)
Comment (Skyhero-admin) [reply index.html:32]: Ohk
I'll fix that

**Review by nhcarrigan — APPROVED** [review]
Looks good to me - we're off to a great start now :D

**Comment by Skyhero-admin** [comment]
Should I add some info about Becca here ?
And can I start adding some styling and images if that's okay

**Comment by nhcarrigan** [comment]
Feel free to add the styling yes!

I don't think we need information for Becca since she's not necessarily a community project.

**Comment by Skyhero-admin** [comment]
ohk

File: index.html [hunk 4]
```diff
@@ -13,13 +13,46 @@
...
+         Participate in our community projects
+      </p>
+      <p class="para">
+         Of course this is a Programming focused community, but that's not all that we're about
+      </p>
+      <p class="para">
+        <b>We also have a Gaming discussion channel on our Discord Server</b>Where our community members stream their Gameplays quite often. We also play together with the other community members quite often.
+      </p>
+      <p class="para">
+        <b>We also have a lot of other fun activity's happenning every now then</b>
```
Comment (starlightknown) [inline index.html:34]: shouldn't it be `activities` and not `activity's` ?

File: index.html [hunk 5]
```diff
@@ -13,13 +13,46 @@
...
+      </p>
+    </section>
+    <section id='goals'>
+      <h2>Our Community Goals and upcoming projects</h2>
+      <p class="para"></p>
+      
+    </section>
+    <section id="whoisNH">
+      <h2>Who is Nhcarrigan ?</h2>
```
Comment (starlightknown) [inline index.html:43]: We can add the same picture of Nick here too like the picture above and then the who is NH part

File: index.html [hunk 6]
```diff
@@ -13,13 +13,46 @@
...
+      <p class="para">I am a web developer focusing on the MEAN stack with a love for TypeScript. Development is a passion of mine, which began with the FreeCodeCamp curriculum on 15 April 2020. In December of 2020 I accepted a position with freeCodeCamp, and have continued to learn, grow, and acquire new skills.
```
Comment (starlightknown) [inline index.html:44]: change `MEAN` to `MERN`

**Review by starlightknown — CHANGES_REQUESTED** [review]

File: index.html (hunk 6 above)
Comment (nhcarrigan) [reply index.html:44]: Nope. I'm a `MEAN` stack dev. 🙂

File: index.html (hunk 5 above)
Comment (nhcarrigan) [reply index.html:43]: We can use this link `https://cdn.nhcarrigan.com/content/profile.jpg`

File: index.html (hunk 4 above)
Comment (nhcarrigan) [reply index.html:34]: Yes.

File: index.html (hunk 6 above)
Comment (starlightknown) [reply index.html:44]: wait that's there? It's MEAN? MEAN and MERN are different?

Comment (nhcarrigan) [reply index.html:44]: MEAN is MongoDB, Express, Angular, and Node
MERN is MongoDB, Express, React, and Node

Comment (starlightknown) [reply index.html:44]: Got it

File: index.html (hunk 5 above)
Comment (Skyhero-admin) [reply index.html:43]: Oh yes
Thanks for pointing out

File: index.html (hunk 4 above)
Comment (Skyhero-admin) [reply index.html:34]: Oh yes,
thanks for pointing out

**Comment by Skyhero-admin** [comment]
Everyone please suggest some colors for the site  😅

**Comment by Skyhero-admin** [comment]
And what kind of image can we use for the as background for #welcome

**Comment by Evavic44** [comment]
> Everyone please suggest some colors for the site 😅
Nhcommunity Colors

**Comment by starlightknown** [comment]
> > Everyone please suggest some colors for the site sweat_smile
> 
> I generated this pallet using the community profile photo as inspiration. What do you all think? sunglasses
> Nhcommunity Colors

cool!

**Comment by starlightknown** [comment]
I think we can add about Nh section liked to the nav bar, here's the PR for it @Skyhero-admin 
#9

**Comment by Skyhero-admin** [comment]
> I think we can add about Nh section liked to the nav bar, here's the PR for it @Skyhero-admin
> #9

oh looks good 👍

**Comment by Skyhero-admin** [comment]
> > Everyone please suggest some colors for the site sweat_smile
> 
> I generated this pallet using the community profile photo as inspiration. What do you all think? sunglasses
> Nhcommunity Colors

thanks for this 👍
I will try adding these

**Comment by Evavic44** [comment]
> > > Everyone please suggest some colors for the site sweat_smile
> > 
> > 
> > I generated this pallet using the community profile photo as inspiration. What do you all think? sunglasses
> > Nhcommunity Colors
> 
> thanks for this 👍

I would like to add it to the `CSS` file as a comment for easy access.

**Comment by Skyhero-admin** [comment]
> > > > Everyone please suggest some colors for the site sweat_smile
> > > 
> > > 
> > > I generated this pallet using the community profile photo as inspiration. What do you all think? sunglasses
> > > Nhcommunity Colors
> > 
> > 
> > thanks for this +1
> 
> I would like to add it to the `CSS` file as a comment for easy access.

Sure that would be cool
You should try adding it

**Comment by Skyhero-admin** [comment]
about nick
Should I keep the background for this section as white since the picture has a white background and it stands out ?
Any ideas

**Comment by nhcarrigan** [comment]
I can re-upload the transparent version to my CDN so it'll fit that colour better.

**Comment by nhcarrigan** [comment]
Hey @Skyhero-admin I gave this a rebase after landing a couple of PRs.

If you're working on this locally, you'll need to update your local files with:
```bash
git fetch origin
git reset --hard origin/main
```

**Comment by Skyhero-admin** [comment]
> s locally, you'll need to update your local files with:

ohk,
Understood

**Comment by Skyhero-admin** [comment]
Some links are yet to be added , should I add them now or can we add them later ?

**Comment by nhcarrigan** [comment]
We can always add things later on - this PR doesn't have to accomplish everything at once. 🙂

**Comment by Skyhero-admin** [comment]
ohk

File: index.html (hunk 6 above)
Comment (Vyvy-vi) [reply index.html]: Nick is MEARN? xD

File: index.html [hunk 7]
```diff
@@ -13,23 +13,76 @@
...
+        We’re a group of members committed to providing a safe, and comfortable space for anyone to try their hand at coding, and have some fun along the way. As our founding member <b><a href="#whoisNH">NhCarrigan</a></b> is committed to helping members of the <b>freecodecamp</b> community, as well as <b>EddieHub</b> by providing further support to any who need it. We have a staff of moderators who share this vision, come meet them here (link) and check out some of their amazing projects!
+      </h3>
+      <h3>
+        You don’t have to be interested in coding to join us, we often have rounds of fun on Among Us, and talk about a variety of topics. Our moderator team with NhCarrigan, work with the community to set goals and provide projects that everyone can contribute to, in order to help build a sense of community
+      </h3>
+    </section>
+      <section id="gaming">
+        <h2>Gaming 🕹️ 🎮 🎲</h2>
+        <p class="para">
+        A love of gaming is something a lot of people share, and we’d love to share it with you! Often playing rounds of Among us, as well as a plethora of other games we are always looking to expand our community of players.
```
Comment (Vyvy-vi) [inline index.html:42]:
```suggestion
        A love for gaming is something a lot of people share, and we’d love to share it with you! Often playing rounds of Among us, as well as a plethora of other games we are always looking to expand our community of players.
```

**Review by Vyvy-vi — COMMENTED** [review]
Maybe we should add some space at bottom of the page?

File: index.html [hunk 8]
```diff
@@ -13,23 +13,76 @@
...
-    </p>
-    <p>
+    <div id="welcome">
+      <h1>Welcome to Carrigan's Community! 🔥</h1>
+    </div>
+    <main>
+    <section id='opening'>
+      <h2>🔥Our community🔥</h2>
+      <h3>
+        We’re a group of members committed to providing a safe, and comfortable space for anyone to try their hand at coding, and have some fun along the way. As our founding member <b><a href="#whoisNH">NhCarrigan</a></b> is committed to helping members of the <b>freecodecamp</b> community, as well as <b>EddieHub</b> by providing further support to any who need it. We have a staff of moderators who share this vision, come meet them here (link) and check out some of their amazing projects!
```
Comment (shreyas44) [inline index.html:33]:
```suggestion
        We’re a group of members committed to providing a safe, and comfortable space for anyone to try their hand at coding, and have some fun along the way. As our founding member <b><a href="#whoisNH">NhCarrigan</a></b> is committed to helping members of the <a href="https://freecodecamp.org"><b>freeCodeCamp</b></a> community, as well as <a href="https://eddiehub.org"><b>EddieHub</b></a> by providing further support to any who need it. We have a staff of moderators who share this vision, come meet them here (link) and check out some of their amazing projects!
```
Add links to FCC and EddieHub?

Comment (Skyhero-admin) [reply index.html:33]: Should I put their website links?

File: index.html (hunk 7 above)
Comment (Skyhero-admin) [reply index.html:42]: Oh
thanks for pointing it out
I had missed it 😅

File: index.html [hunk 9]
```diff
@@ -13,23 +13,76 @@
...
-      <li><a href="#discussions">Discussions</a></li>
-      <li><a href="./views/calendar.html">Goals & Events</a></li>
-      <li><a href="#community">Community</a></li>
+      <li><a href="#goals">Goals</a></li>
+      <li><a href="#discussion">Discussions</a></li>
+      <li><a href="./views/calendar.html">Events</a></li>
+      <li><a href="#opening">Community</a></li>
       <li><a href="#becca">Becca</a></li>
-      <li><a href="#about">About NhCarrigan</a></li>
+      <li><a href="#whoisNH">About NhCarrigan</a></li>
```
[the remark this answers is not in the document]
Comment (Skyhero-admin) [reply index.html:24]: Ohk I will fix that

File: index.html (hunk 8 above)
Comment (shreyas44) [reply index.html:33]: Yep!

Comment (Skyhero-admin) [reply index.html:33]: ohk 👍

**Comment by Skyhero-admin** [comment]
should I fix something else ?

File: index.html [hunk 10]
```diff
@@ -13,23 +13,78 @@
...
+        <p class="para">
+          <!--some links are yet to be added in this section-->
+        Welcome to our Discussion Center, before joining our community please make sure you’ve read and agreed to our <b><a href="https://handbook.nhcarrigan.com/#/pages/rules">“Rules”</a></b>, if you’re looking for help with a particular freecodecamp problem we have “Resources”(link), and if you have any questions with anything we have a very supportive community that is happy to answer anyquestions in our “Help”(link) section.
+      </p>
+      </section>
+      
+    </section>
+    <section id='goals'>
+      <h2>Our Community Goals and upcoming projects</h2>
+      <p class="para">Here at the NhCommunity we like to have some fun! We are often working on a handful of projects to improve our community as a whole, that everyone is welcome to contribute to.</p>
```
Comment (DennaGrey) [inline index.html:59]:
```suggestion
      <p class="para">Here at the nhcommunity we like to have some fun! We are often working on a handful of projects to improve our community as a whole, that everyone is welcome to contribute to.</p>
```

**Review by DennaGrey — DISMISSED** [review]
Just a tiny grammar thing :)

File: index.html (hunk 10 above)
Comment (Skyhero-admin) [reply index.html]: ohh
I will fix it

**Review by nhcarrigan — APPROVED** [review]
This looks good, and has been open for a while. Landing it.

**Pull request merged**